src/legacy/ui/public/agg_types/__tests__/buckets/significant_terms.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from '@kbn/expect';
+import ngMock from 'ng_mock';
+import { aggTypes } from '../..';
+
+describe('Significant Terms Agg', function () {
+
+  describe('order agg editor UI', function () {
+
+    describe('convert include/exclude from old format', function () {
+
+      let $rootScope;
+
+      // function init({ responseValueAggs = [], aggParams = {} }) {
```

Comment:
I think as you mentioned that can all be removed, since it's not necessary for those specific tests